CONTRIBUTING.md

```diff
@@ -20,10 +20,14 @@ applies to other repos as well.)
 1. [Create a fork of the numberscope/frontscope repo](./doc/working-with-git-and-github.md#create-a-fork).
 2. [Clone your fork of numberscope/frontscope](./doc/working-with-git-and-github.md#clone-a-repo).
 3. [Create a branch for your contribution](./doc/working-with-git-and-github.md#create-a-branch).
-4. [Read about basic Git operations](./doc/working-with-git-and-github.md#basic-git-operations).
+4. If you are unfamilar with them or would like a refresher,
+   [read about basic Git operations](./doc/working-with-git-and-github.md#basic-git-operations).
 5. [Push your branch to GitHub](./doc/working-with-git-and-github.md#push-a-branch).
 6. [Read Numberscope's coding principles guide](./doc/code-principles.md).
-7. [Read Numberscope's style guide](./doc/code-style.md).
+7. [Read Numberscope's style guide](./doc/code-style.md). You should also
+   familiarize yourself with frontscope's internal code structure and APIs;
+   the documentation available on these topics is listed in the "Internal code
+   and APIs" section of the navigation bar to the left.
```

Comment:
At the end of line 30, I suggest adding:

"If you're reading this on GitHub, you can view this documentation on our website here: \<link\>. Alternatively, if you'd prefer to stay on GitHub, you can view the documentation in the source code here: \<link\>."

Reply:
(Good ideas, you can see how I worked this in and let me know what you think.)